### PR TITLE
Improved error message for multiple Dropbox add-ons [#OSF-5744]

### PR DESCRIPTION
**Pull request by baylee-d**
## Purpose

When a user adds multiple Dropbox add-ons, the error message that displayed was unhelpful.  The original message  was: 

> Error while authorizing add-on. Please log in to your Dropbox account and grant access to the OSF to enable this addon.

This didn't give a lot of help to the user to set up a second Dropbox account.  The purpose of this PR was to fix the message so that it would be of more help to the user.  

## Changes

The error only shows when the user tries to add an additional account without logging out of their current Dropbox account.  Therefore the message that was displayed was changed to be more accurate to the steps needed to fix the issue.  The issue that displays now shows:

screen shot 2016-06-29 at 1 44 33 pm

## Side effects

There shouldn't be any side effects from the fixes applied.  

## Ticket

https://openscience.atlassian.net/browse/OSF-5744

### CODE REVIEW

File: website/static/js/addonSettings.js [hunk 1]
```diff
@@ -92,7 +92,9 @@ var OAuthAddonSettingsViewModel = oop.defclass({
             self.updateAccounts().done( function() {
                 (self.accounts().length > accountCount) ?
                     self.setMessage('Add-on successfully authorized. To link this add-on to an OSF project, go to the settings page of the project, enable ' + self.properName + ', and choose content to connect.', 'text-success') :
-                    self.setMessage('Error while authorizing addon. Please log in to your ' + self.properName + ' account and grant access to the OSF to enable this addon.', 'text-danger');
+                    (self.name == 'dropbox') ?
```
Comment (brianjgeiger) [inline website/static/js/addonSettings.js]: Since it's already multi-line, I'd go with a standard if-else block here.

**Comment by brianjgeiger** [comment]
Minor change requested, otherwise looks good. 🐧

**Comment by brianjgeiger** [comment]
Travis CI says you have a jshint error. "/home/travis/build/CenterForOpenScience/osf.io/website/static/js/addonSettings.js: line 96, col 37, Expected '===' and instead saw '=='."

**Comment by brianjgeiger** [comment]
Pass done. The if structure looks much better. Get the tests passing and this should be ready to go. 🐧

**Comment by sloria** [comment]
Fixed error, squashed, and merged. 👍

**Pull request closed without merging**